web/packages/teleport/src/services/auth/makeMfa.ts

```diff
@@ -146,6 +141,11 @@ export function makeWebauthnAssertionResponse(res): WebauthnAssertionResponse {
   };
 }
 
+export type SsoChallengeResponse = {
+  requestId: string;
+  token: string;
```

Comment:
Which party generates this token?

Reply:
We generate it in the backend when asking for challenges, returned in the same response that a webauthn challenge from us would come from.